spec/attribute.dd

```diff
@@ -294,15 +311,26 @@ static assert(S.sizeof == 32);
         byte boundaries.
         )
 
-        $(UNDEFINED_BEHAVIOR If any pointers and references to GC
-        allocated objects are not aligned on `size_t` byte boundaries.)
+$(SPEC_RUNNABLE_EXAMPLE_FAIL
+---
+struct S
+{
+  align(1):
```

Comment:
```suggestion
align(1):
```

Reply:
I was following the style of the other align examples. I also noticed two space indent for another example with `private:`.